middleware/auth-utils/config.js

```diff
@@ -122,7 +122,7 @@ Config.prototype.configure = function configure (config) {
    * Authentication server URL
    * @type {String}
    */
-  this.authServerUrl = resolveValue(config['auth-server-url'] || config['server-url'] || config.serverUrl || config.authServerUrl);
+  this.authServerUrl = (resolveValue(config['auth-server-url'] || config['server-url'] || config.serverUrl || config.authServerUrl) || "").replace(/\/$/, "");
```

Comment:
@mhajas Just a stupid question, but why's there ` || ""`? Do we expect `resolveValue` to return some falsy value?

Reply:
I was surprised too, but it may return undefined. It happens in this test: https://github.com/keycloak/keycloak-nodejs-connect/blob/master/test/grant-manager-spec.js#L18